src/containers/LearningResourcePage/components/LearningResourceForm.jsx

```diff
@@ -310,7 +312,11 @@ class LearningResourceForm extends Component {
         ),
       },
     ];
-    if (model.id) {
+
+    if (
+      model.id &&
+      userAccess.includes(`taxonomy-${config.ndlaEnvironment}:write`)
```

Comment:
Denne mister snart miljø-delen av verdien. Slik at det blir taxonomy:write. Er lurt om det kan tas høgde for også.